server/api/rounds.py

```diff
@@ -38,14 +38,16 @@ def get_current_round(election: Election) -> Optional[Round]:
     return next(iter(rounds), None)
 
 
-def is_audit_complete(round_id: str):
-    targeted_round_contest = (
-        RoundContest.query.filter_by(round_id=round_id)
+def is_audit_complete(round: Round):
+    if not round.ended_at:
+        return None
```

Comment:
I realize None acts like false, but why reason not to return actual `False`?

Reply:
The way this value has functioned so far is to have three distinct states:
- `None` - a round is in progress
- `true` - the round is over and the audit is complete
- `false` the round is over and another round is needed

This change preserves this behavior. Open to feedback as to whether it's good to have all three states or not. I think when I originally designed it my thinking was that you should only ask whether the audit is complete once a round has ended, so there should be no answer during a round.